config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
   namespace :api do
+    get 'products/index'
+  end
```

Comment:
What does that do? not sure if you need it

Reply:
It was auto generated by Rails, but we have removed it now.